roles/dns/defaults/main.yml

```diff
@@ -1,6 +1,5 @@
 ---
-algo_local_dns: false
-listen_port: "{% if algo_local_dns %}5353{% else %}53{% endif %}"
+algo_dns_adblocking: false
 # the version used if the latest unavailable (in case of Github API rate limited)
 dnscrypt_proxy_version: 2.0.10
```

Comment:
Most recent version is [2.0.25](https://github.com/jedisct1/dnscrypt-proxy/blob/master/ChangeLog), and current version on my Lightsail instance is 2.0.23.